src/app/services/ws.service.ts

```diff
@@ -139,6 +141,13 @@ export class WebSocketService {
     this.eventSubscribers.clear();
   }
 
+  getDebouncedWebSocketStream$(): Observable<unknown> {
```

Comment:
This service doesn't know why 5 seconds is needed, thus it shouldn't be part of this service.